src/transformers/models/swin/modeling_swin.py

```diff
@@ -190,17 +348,30 @@ def __init__(self, input_resolution, dim, norm_layer=nn.LayerNorm):
         self.reduction = nn.Linear(4 * dim, 2 * dim, bias=False)
         self.norm = norm_layer(4 * dim)
 
-    def forward(self, input_feature):
-        height, width = self.input_resolution
+    def maybe_pad(self, input_feature, width, height):
```

Comment:
The maybe_pad method in `SwinPatchEmbeddings` take height, width whereas here it's width, height.

Let's use a consistent API